### DISCUSSION TITLE
Rocket v0.5.0-rc.2 is available!

### DISCUSSION THREAD

**Discussion by SergioBenitez** [rwf2/Rocket#2165] [Announcements]
See the [CHANGELOG](https://github.com/SergioBenitez/Rocket/blob/v0.5.0-rc.2/CHANGELOG.md), [news article](https://rocket.rs/v0.5-rc/news/2022-05-09-version-0.5-rc.2/), [migration guide](https://rocket.rs/v0.5-rc/guide/upgrading-from-0.4/), and [FAQ](https://rocket.rs/v0.5-rc/guide/faq/) for more information.

<hr /><em>This discussion was created from the release <a href='https://github.com/SergioBenitez/Rocket/releases/tag/v0.5.0-rc.2'>Rocket v0.5.0-rc.2</a>.</em>

**Comment by TaeyoonKwon** [comment]
CHANGELOG is returning not found

**Reply by nicoburns** [reply]
Should be https://github.com/SergioBenitez/Rocket/blob/master/CHANGELOG.md

**Reply by SergioBenitez** [reply]
Thanks! This is a GitHub bug, but I've fixed the link now.

**Comment by Cryt1c** [comment]
What's the current plan for the full release of 0.5? On https://rocket.rs/v0.5-rc/news/2022-05-09-version-0.5-rc.2/ it's said to be released in late May 2022.

**Reply by nicoburns** [reply]
It was also said to be released in June 2021 (https://rocket.rs/v0.5-rc/news/2021-06-09-version-0.5-rc.1/). I wouldn't hold your breath.

**Reply by moha-verse** [reply]
Right, we all waiting for it. :/

**Comment by kragil** [comment]
It seems that that the project never recovered from its hiatus. Commits and contributions are way down. There are months without any commits. 

I don’t feel comfortable basing any serious project on this. It feels abandoned. 
I’ll watch if a 0.5 release will breath new life into this, but for now other rust web frameworks seem to have all the traction.
